uavcan/primitive/ArrayOfBits.1.0.uavcan

```diff
@@ -0,0 +1,5 @@
+#
```

Comment:
Can we make it so we don't need these silly definitions? Can we, for instance, instead allow primitive types (and arrays) to be sent on a subject ID as well as composite definitions?

Reply:
I beg to differ. The definitions may look silly, but they are architecturally sound, unless I am mistaken somewhere. They explicitly define the set of basic types that can be exchanged over the bus, avoiding possible excessive fragmentation if one were to use oddly sized types like `uint51`. They operate within the framework of existing type system, not needing additional provisions from the specification, keeping it simple. They would be required anyway if we were to add timestamped primitives.

I don't see a compelling reason to add more complexity to the specification.